addons/bestja_requests/models.py

```diff
@@ -195,14 +195,14 @@ class Request(models.Model):
         ondelete='restrict',
     )
     items = fields.One2many('bestja_requests.item', inverse_name='request')
-    comments = fields.Text(string="uwagi")
+    comments = fields.Text(string="Uwagi")
     manager = fields.Many2one(
         'res.users',
         string="Menadżer projektu",
         related='project.responsible_user',
     )
     manager_phone = fields.Char(
-        string="Telefon menadżera",
+        string="Dane kontaktowe menadżera",
```

Comment:
Changing a field name would be confusing, since it is a related field pointing directly to manager's phone number.